src/App.jsx

```diff
@@ -6,24 +6,39 @@ import RestaurantList from "./components/restaurant/RestaurantList/RestaurantLis
 import RestaurantDetailModal from "./components/modal/RestaurantDetailModal";
 import AddRestaurantModal from "./components/modal/AddRestaurantModal";
 import getFilteredRestaurant from "./utils/getFilteredRestaurant";
+import useModal from "./hooks/useModal";
 
 function App() {
   const [selectedCategory, setCategory] = useState("전체");
+  const restaurantDetailModal = useModal(false);
```

Comment:
지금 방식으로 사용하는 것도 어색하진 않습니다. 그러나 일반적으로는 구조분해할당을 사용해요. 

```suggestion
  const { openModal: openAddRestaurantModal, ... } = useModal(false);
```

Reply:
그렇군요! 두 개의 모달에 적용해 보겠습니다. 👍